controller/lib/core/actuators.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef ACTUATORS_H
+#define ACTUATORS_H
+
+#include "blower_fsm.h"
+#include "hal.h"
+
+typedef struct _ActuatorsState {
+  ValveState expire_valve_state;
+  float fan_power;
+} ActuatorsState;
+
+#define ActuatorsState_init_zero                                               \
+  { ValveState::OPEN, 0.0 }
+
+ActuatorsState actuators_execute(ActuatorsState desired_state);
```

Comment:
Why doesn't this return void?

Reply:
In my eyes, but maybe I am getting a bit ahead, once we have an actuators_state field inside ControllerStatus, we can call actuators_execute to update ControllerStatus.actuators_state only when the actuators have indeed been activated.

Reply:
Hm, what  if we said that actuators_execute always applies the actuators, and if you don't want to do that, then you shouldn't call it?

That seems like the minimal amount of logic we could put in here.  Which is good, because then the logic of "do we update the actuators" can be modeled in Modelica.

(If we do that, then presumably this should return void.)

Reply:
Agreed, we can always change our minds when the time comes.